### PR TITLE
Update Basic-cloud-server-security with script.md

### PR DESCRIPTION
**Pull request by kylelaffoon**
Add link to Everett's script in the _Includes files.

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-72152b324f/how-to/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-9488561a6c/how-to/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-9488561a6c/how-to/).

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-106c9d36fb/how-to/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-8f32128805/how-to/).

**Comment by cyrichardson** [comment]
@kylelaffoon 
I checkout out the staging preview for this change at:

https://staging.developer.rackspace.com/support.rackspace.com/build-8f32128805/how-to/basic-cloud-server-security/

I'm not sure that this is what you are expecting to see. It seems to need at least more introductory material. Also, there is no "code" formatting applied to anything included from the .sh file.  (Maybe you can only get that type of formatting by not using the include, but rather copying the file into the article and applying formats there. I'm not sure about that though.)

Please check the link above and let me know if you want to go forward with this PR merge or if you'd like to make further changes.

Thanks!

File: content/cloud-servers/basic-cloud-server-security.md [hunk 1]
```diff
@@ -14,382 +14,7 @@ product_url: cloud-servers
...
-
-If you have trouble making a new connection after restarting ssh, check
-the symptoms to see what may be wrong. If the connection times out,
-there may be a problem with the iptables config. If you get a warning
-about a private key, your key may not be installed on the server
-properly (check for extra linebreaks or characters that were missed in a
-copy and paste operation). If you've been rebuilding the server then
-you may need to [remove the host key from your known hosts file](/how-to/rackspace-cloud-essentials-checking-a-server-s-ssh-host-fingerprint-with-the-web-console)
-before you can make a connection.
+{% include cloud-servers/basic-cloud-server-security/secure.sh %}
```
Comment (everett-toews) [inline content/cloud-servers/basic-cloud-server-security.md]: Surround this with `<pre>` and `<code>` like so

```
<pre><code>
{% include cloud-servers/basic-cloud-server-security/secure.sh %}
</code></pre>
```

I believe CSS should take care of the rest. If not, we might have to pester @ktbartholomew.

And @cyrichardson is right, the script needs more introductory material.

Thanks!

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-43453a78ca/how-to/)

File: content/cloud-servers/basic-cloud-server-security.md [hunk 2]
```diff
@@ -4,392 +4,25 @@ node_id: 268
...
-**Note**: These are the basics of connecting to a Linux Cloud Server and
-setting up security.
+Run the following script when setting security for your cloud server. This script 
+is only intended to be more secure than the default configuration. No promises are 
+made about this script preventing your server from being attacked. Ensure that you 
+are writing secure application code.
 
-### Previous section
+**Note:** This script assumes you're running it initially as root and logged in using
+a key pair. If you didn't, you'll be locked out of your VM.
```
Comment (everett-toews) [inline content/cloud-servers/basic-cloud-server-security.md]: Should link to a how to on creating a server with a key pair here.

File: content/cloud-servers/basic-cloud-server-security.md [hunk 3]
```diff
@@ -4,392 +4,25 @@ node_id: 268
...
+created_by: Everett Toews
+last_modified_date: '2016-04-11'
+last_modified_by: Kyle Laffoon
 product: Cloud Servers
 product_url: cloud-servers
 ---
 
-**Note**: These are the basics of connecting to a Linux Cloud Server and
-setting up security.
+Run the following script when setting security for your cloud server. This script
```
Comment (everett-toews) [inline content/cloud-servers/basic-cloud-server-security.md]: "...when setting up security for an Ubuntu cloud server."

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-68599c0572/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-7a32d6ae4d/how-to/)

**Comment by cyrichardson** [comment]
LGTM - Merging

**Pull request merged**